apps/web/components/Webreader/Webreader.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC, useEffect, useState } from 'react'
+import { useRouter } from 'next/router'
+import { useI18n } from '@island.is/web/i18n'
+import { Box, ResponsiveProp, Space } from '@island.is/island-ui/core'
+import { CUSTOMER_ID, SCRIPT_URL } from './config'
+
+declare global {
+  interface Window {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    rsConf: any
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+declare const ReadSpeaker: any
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+declare const rspkr: any
+interface WebReaderProps {
+  readId?: string
+  readClass?: string
+  marginTop?: ResponsiveProp<Space>
+  marginBottom?: ResponsiveProp<Space>
+}
+
+export const Webreader: FC<WebReaderProps> = ({
+  readId = 'main-content',
+  readClass,
+  marginTop = 3,
+  marginBottom = 3,
+}) => {
+  const [href, setHref] = useState('')
+  const router = useRouter()
+  const { activeLocale } = useI18n()
+
+  useEffect(() => {
+    const routeChangestart = (url) => {
```

Comment:
We do this on routeChanges since the Reader doesn't handle dynamic page changes well, in fact, it breaks otherwise